### PR TITLE
Moved collection view extensions to separate UI file

### PR DESCRIPTION
**Pull request by peytonshetler**
## Summary
I had to extend the HomeScreen View controller with some collection view stuff.  I moved it into a separate UI file.  I think that's where it goes since it directly affects the the UI?  Haha.

#### Characteristics

- [] Feature completed
- [X] Experimental
- [X] UI affected (screen shots included)

### CODE REVIEW

File: Been-There/Been-There/Controller/LandingScreen.swift [hunk 1]
```diff
@@ -56,6 +56,9 @@ class LandingScreen: UIViewController {
                 return
```
Comment (JZDesign) [inline Been-There/Been-There/Controller/LandingScreen.swift]: Sorry I didn't think of this earlier...

Naming conventions: Consider renaming to `LandingScreenViewController`

| Group | Super Class | 
|----|----|
| LandingScreen | ViewController |

File: Been-There/Been-There/Controller/RegisterScreen.swift [hunk 2]
```diff
@@ -39,6 +39,10 @@ class RegisterScreen: UIViewController {
         dismiss(animated: true, completion: nil)
```
Comment (JZDesign) [inline Been-There/Been-There/Controller/RegisterScreen.swift]: Sorry I didn't think of this earlier...

Naming conventions: Consider renaming to `RegistrationScreenViewController`

| Group | Super Class | 
|----|----|
| RegistrationScreen | ViewController |

File: Been-There/Been-There/View/Cell+HomeScreen.swift [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+//  Cell+HomeScreen.swift
+//  Been-There
+//
+//  Created by Peyton Shetler on 3/28/19.
+//  Copyright © 2019 Peyton Shetler. All rights reserved.
+//
+
+import UIKit
+
+class CellForHomeScreen: UICollectionViewCell {
```
[the remark this answers is not in the document]
Comment (peytonshetler) [reply Been-There/Been-There/View/Cell+HomeScreen.swift]: Annnd done!

File: Been-There/Been-There/Controller/HomeScreen.swift [hunk 4]
```diff
@@ -0,0 +1,35 @@
+//
+//  HomeScreen.swift
+//  Been-There
+//
+//  Created by Peyton Shetler on 3/25/19.
+//  Copyright © 2019 Peyton Shetler. All rights reserved.
+
+import UIKit
+
+class HomeScreen: UIViewController {
```
[the remark this answers is not in the document]
Comment (peytonshetler) [reply Been-There/Been-There/Controller/HomeScreen.swift]: Done!

File: Been-There/Been-There/Controller/UI+HomeScreen.swift [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+//  UI+HomeScreen.swift
+//  Been-There
+//
+//  Created by Peyton Shetler on 3/28/19.
+//  Copyright © 2019 Peyton Shetler. All rights reserved.
+//
+
+import UIKit
+
+extension HomeScreen: UICollectionViewDataSource, UICollectionViewDelegate, UICollectionViewDelegateFlowLayout {
```
[the remark this answers is not in the document]
Comment (peytonshetler) [reply Been-There/Been-There/Controller/UI+HomeScreen.swift]: Done

File: Been-There/Been-There/Controller/RegisterScreen.swift (hunk 2 above)
Comment (peytonshetler) [reply Been-There/Been-There/Controller/RegisterScreen.swift]: Done

File: Been-There/Been-There/Controller/LandingScreen.swift (hunk 1 above)
Comment (peytonshetler) [reply Been-There/Been-There/Controller/LandingScreen.swift]: Also done

File: Been-There/Been-There/Controller/LandingScreen.swift [hunk 6]
```diff
@@ -56,6 +56,9 @@ class LandingScreen: UIViewController {
...
             }
             print("User logged in!") // Login successful.  This print is a placeholder
+            self.dismiss(animated: false, completion: nil)
+            let homeScreen = HomeScreen()
+            self.present(homeScreen, animated: true, completion: nil)
```
[the remark this answers is not in the document]
Comment (peytonshetler) [reply Been-There/Been-There/Controller/LandingScreen.swift]: Just tried this and it isn't working for some reason.  It builds fine, but in simulator when it should present the HomeScreen it just...doesn't.  I'll do some research.

**Pull request merged**